packages/remix-dev/compiler/css/bundle.ts

```diff
@@ -45,16 +45,45 @@ export let write = async (ctx: Context, outputFiles: esbuild.OutputFile[]) => {
         await fse.writeFile(asset.path, asset.contents);
       }),
   ]);
-};
+}
 
-export let isBundle = (
+function isCssBundleFile(
   ctx: Context,
   outputFile: esbuild.OutputFile,
   extension: ".css" | ".css.map"
-): boolean => {
+): boolean {
   return (
     path.dirname(outputFile.path) === ctx.config.assetsBuildDirectory &&
     path.basename(outputFile.path).startsWith("css-bundle") &&
     outputFile.path.endsWith(extension)
   );
-};
+}
+
+interface GroupedCssBundleFiles {
```

Comment:
This can just be a `type` I guess?

```suggestion
type GroupedCssBundleFiles = {
```